app/assets/stylesheets/12-artist-show.scss

```diff
@@ -55,7 +56,43 @@
     padding: 0px 0px 0px 30px;
     gap: 25px;
 }
+// PlaylistNav
+.playlist-nav, .album-nav {
+    display: flex;
+    background: #23201be6;
+    width: auto;
+    align-items: center;
+    padding: 25px 35px;
+    gap: 25px;
+}
+
+.artist-dropdown {
+    left: 145px;
+    top: 365px;
+    align-items: flex-start;
+    min-width: 140px;
+}
+
+.playlist-dropdown-button {
+    width: 100%;
+    font-size: 10.5pt;
+    font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande", "Lucida Sans", Arial, sans-serif;
+    cursor: pointer;
+    color: white;
+    padding: 11px 11px 11px 11px;
```

Comment:
nit:` padding: 11px` should do the trick as well - not a recommendation either way, just sharing in case

Reply:
Yes I did know this 😓 .... This must be really old code. Thanks for nitpicking.